features/support/publishing_api.rb

```diff
@@ -0,0 +1,5 @@
+Before do
+  stub_any_publishing_api_put_content
+  stub_any_publishing_api_publish
+  stub_any_publishing_api_unpublish
+end
```

Comment:
I'm not a fan of this sort of global approach and would prefer to see this done explicitly if possible.

Reply:
I've moved the stubbing into features/support/recommended_links.rb. What do you think? https://github.com/alphagov/search-admin/pull/97/commits/fe7ef72e5bcce61b0ca6adb565d53cf30101affc#diff-6bdf893696b0ca8b8be584508e81540cR2

It still uses the generic `stub_any...` methods. It's tricky be more specific because you need to know the content IDs to stub the whole request. I also think that level of testing is more suited to an integration test than a Cucumber test.